build/includes/sdk.mk

```diff
@@ -0,0 +1,93 @@
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+#         ____  ____   ____   _____           _ _	
+#    __ _|  _ \|  _ \ / ___| |_   _|__   ___ | (_)_ __   __ _	
+#   / _` | |_) | |_) | |       | |/ _ \ / _ \| | | '_ \ / _` |	
+#  | (_| |  _ <|  __/| |___    | | (_) | (_) | | | | | | (_| |	
+#   \__, |_| \_\_|    \____|   |_|\___/ \___/|_|_|_| |_|\__, |	
+#   |___/                                               |___/ 
+
+build_sdk_base_tag = agones-build-sdk-base:$(build_version)
```

Comment:
If the Dockerfile from the sdk tooling gets changed - does that force a rebuild? (should it?)

Reply:
it should, and it does not... I'm looking into it.

Reply:
I went for the easy path, rebuilding images all the times.

Reply:
Sounds like a good first start, we can iterate as we need :+1: